specs/solid-indexing/index.html

```diff
@@ -44,7 +86,7 @@ <h3>Terminology</h3>
 
       <p>
         <dfn data-lt="indexes">Index</dfn>
-        A list of keywords with pointers to where further information about the keyword is found. An index summarizes or
+        A list of keyword with pointers to where further information about the keyword is found. An index summarizes or
         characterizes the content and its location.
```

Comment:
Why remove the plural??
Anyway, I didn't notice this definition before, but I don't find it satisfying. The notion of keyword does not match what we are doing (or only partially).

I can not make a suggestion here (this impacts non-modified lines), I will make a PR to the PR.

Reply:
This definition was the one used in our different presentations (slides).

Reply:
Fixed in https://github.com/Wimmics/solid-start/pull/47/commits/7c73164dd34d8aaddd7e8e64ca93e83bae74f4e2